### DISCUSSION TITLE
ABACUS v3.2 to be released on Apr 1, 2023

### DISCUSSION THREAD

**Discussion by ouqi0711** [deepmodeling/abacus-develop#1833] [General]
ABACUS v3.2 is scheduled to be released on April 1, 2023. We plan to offer the following improvements:

## Features & Bug fixes
- Inclusion of the interface with [Hefei-NAMD](https://github.com/QijingZheng/Hefei-NAMD), an ab-initio nonadiabatic molecular dynamics (NAMD) program that enables ab initio NAMD investigations on excited carriers in condensed matter systems.
- Inclusion of the toolkit for the analysis of molecular dynamics results. 
- New features and optimized algorithms are merged in AI-aided methodologies, including DeepH, DeePKS, and DP-GEN. 

## Improved User Experience 
- Default values of various input parameters as well as the output formats of some properties are optimized, aiming for an improved user experience in this new version.

## Performance Optimization
- Parallel efficiency using OpenMP is further improved in cooperation with SenseTime HPC team. 

## Autotest Workflow and Code Coverage
- The performance of ABACUS is now monitored by an autotest workflow.
- Code coverage in this new version is 60%.

**Comment by Satinelamp** [comment]
Isn't the TDDFT method in ABACUS fit for condensed matter systems?

**Reply by ouqi0711** [reply]
In principle, it can be applied to condensed matter systems. However, a velocity gauge implementation is required in such case, which may not be accomplished in v3.2. Therefore, we only include the rt-TDDFT feature for molecular/cluster system in this version.
